api/projects/projects.go

```diff
@@ -32,3 +36,47 @@ func create(c echo.Context) error {
 
 	return c.String(http.StatusOK, "Created project: "+projectName)
 }
+
+func list(c echo.Context) error {
+	db, ok := c.Get(constants.CTX_DB).(*gorm.DB)
+	if !ok {
+		logrus.WithFields(logrus.Fields{
+			"context": c,
+		}).Error("Unable to get DB from context in project create")
+		return errors.New("Unable to get DB from context")
+	}
+
+	projects := db.Find(&[]models.Project{})
+	return c.JSON(http.StatusOK, projects)
+}
+
+func show(c echo.Context) error {
+	db, ok := c.Get(constants.CTX_DB).(*gorm.DB)
+	if !ok {
+		logrus.WithFields(logrus.Fields{
+			"context": c,
+		}).Error("Unable to get DB from context in project create")
+		return errors.New("Unable to get DB from context")
+	}
+
+	id := c.Param("id")
```

Comment:
if id is empty, it would use the list route which shows everything. I'm gonna add handling for if `id` is invalid. What currently happens with gorm is that a `Project` with `ID = 0` is returned.